src/main/java/picard/cmdline/CommandLineProgram.java

```diff
@@ -243,21 +244,32 @@ public int instanceMain(final String[] argv) {
             System.setProperty("java.io.tmpdir", f.getAbsolutePath()); // in loop so that last one takes effect
         }
 
+        PathHelper.initilizeAll();
+
         if (!QUIET) {
             System.err.println("[" + new Date() + "] " + commandLine);
 
             // Output a one liner about who/where and what software/os we're running on
             try {
+                StringBuilder pathProvidersBuilder = new StringBuilder();
+
+                for(PathHelper.PathProviders providers : PathHelper.PathProviders.values()) {
+                    pathProvidersBuilder.append(String.format("Provider %s is%s available; ",
+                            providers.name(), providers.isAvailable() ? "" : " not"));
+                }
+
+
                 final boolean usingIntelDeflater = (BlockCompressedOutputStream.getDefaultDeflaterFactory() instanceof IntelDeflaterFactory &&
                         ((IntelDeflaterFactory)BlockCompressedOutputStream.getDefaultDeflaterFactory()).usingIntelDeflater());
                 final boolean usingIntelInflater = (BlockGunzipper.getDefaultInflaterFactory() instanceof IntelInflaterFactory &&
                         ((IntelInflaterFactory)BlockGunzipper.getDefaultInflaterFactory()).usingIntelInflater());
                 final String msg = String.format(
-                    "[%s] Executing as %s@%s on %s %s %s; %s %s; Deflater: %s; Inflater: %s; Picard version: %s",
+                    "[%s] Executing as %s@%s on %s %s %s; %s %s; Deflater: %s; Inflater: %s; %sPicard version: %s",
```

Comment:
Can you add a space before `Picard` and modify how you build the path providers string not include the last one?  Optional.